app/views/task/index.html.erb

```diff
@@ -1,26 +1,47 @@
-<header><h1>Reading List</h1></header>
-
-<p class="mininav">
-    <%= link_to "Add a book", "/tasks/new" %>
-</p>
-
-<table>
-  <tr class="header">
-  <td>Title</td>
-  <td>Status</td>
-  <td>Unshelve it?</td>
-  </tr>
-
-  <% @displaytasks.each do |task| %>
-    <tr>
-      <td><%= link_to "#{task[:taskname]}", "/tasks/#{task[:id]}" %></td>
-      <td><%= task[:comp_status] %></td>
-      <td><%= link_to "Delete", "/tasks/#{task[:id]}", method: :delete, data: { confirm: "Are you sure?" } %></td>
+<header class = "header">
```

Comment:
We want to be sure to put spaces on either side of the equals sign in our Ruby code but not in our HTML